includes/admin/class-sensei-extensions.php

```diff
@@ -52,7 +52,7 @@ public function enqueue_admin_assets() {
 		$screen = get_current_screen();
 		if ( in_array( $screen->id, [ 'sensei-lms_page_sensei-extensions' ], true ) ) {
 			Sensei()->assets->enqueue( 'sensei-extensions', 'extensions/index.js', [], true );
-			Sensei()->assets->enqueue( 'sensei-extensions-style', 'extensions/extensions.css' );
+			Sensei()->assets->enqueue( 'sensei-extensions-style', 'extensions/extensions.css', [ 'sensei-wp-components' ] );
```

Comment:
I think this can be just `wp-components`, as it's only needed for the `Snackbar` styles, right? Which don't have any Sensei color overrides.

Reply:
> I wonder if we just bring in the Button from @wordpress/components to deal with everything.

I used this approach you suggested, which helped to improve a lot of code. =)
So now it's really needed.